arduino-ide-extension/src/browser/theia/terminal/terminal-widget-impl.ts

```diff
@@ -0,0 +1,23 @@
+import { injectable } from '@theia/core/shared/inversify';
+import { TerminalWidgetImpl as TheiaTerminalWidgetImpl } from '@theia/terminal/lib/browser/terminal-widget-impl';
+import debounce from 'p-debounce';
```

Comment:
Thanks for noticing it; I did not. It's because the change is the "cherry-pick" of the upstream fix, which uses `p-debounce`. See here: https://github.com/eclipse-theia/theia/pull/12587/commits/6ab16d5b70a9f318ab398bece1095a152ebf0223#diff-da3ea75041150eb2403247c5ed84542210774c326c0e3b334029f4e3afb51fc0R45.

Reply:
Related: https://github.com/eclipse-theia/theia/issues/9139